elements/rh-back-to-top/rh-back-to-top.ts

```diff
@@ -0,0 +1,147 @@
+import { LitElement, html, type PropertyValues } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+import { Logger } from '@patternfly/pfe-core/controllers/logger.js';
+
+import '@patternfly/elements/pf-icon/pf-icon.js';
+
+import styles from './rh-back-to-top.css';
+
+
+/**
+ * Back to top component is a fragment link that allows users to quickly navigate
+ * to the top of a lengthy content page.
+ *
+ * @summary A shortcut to the top of the page content
+ *
+ * @csspart trigger - back to top link anchor element
+
+ * @slot - Text for the back to top link
+ *
+ */
+@customElement('rh-back-to-top')
+export class RhBackToTop extends LitElement {
+  static readonly styles = [styles];
+
+  /** Flag to always show back to top button, defaults to false. */
+  @property({ reflect: true, type: Boolean, attribute: 'always-visible' }) alwaysVisible = false;
```

Comment:
Maybe I'm being overly paranoid, but what if this was `visibility="always"` or some such?

Reply:
We can change it, this code right now just mirrors `pf-back-to-top`.    I'm trying to think if there would ever be any other options... but that would leave an open door there if needed.